packages/tools/src/tools/annotation/CircleROITool.ts

```diff
@@ -1021,8 +1023,15 @@ class CircleROITool extends AnnotationTool {
         stdDev /= count;
         stdDev = Math.sqrt(stdDev);
 
+        const { suvbw } =
```

Comment:
we don't need to do this metadata query on each cachedStats calculation since it doesn't change with annotation data change, it can be done outside, in the render loop, just before _getTextLines calls

Reply:
moved the metadata query